### PR TITLE
Firelock Unweld Grammar

### PR DESCRIPTION
**Pull request by kugamo**
## What Does This PR Do

Changes the dialogue when you unweld a firelock to say "unweld" instead of weld again.

## Why It's Good For The Game

Bad grammar bad

## Changelog
:cl:
spellcheck: Firelock unweld message now says "unweld" instead of "weld".
/:cl:

### CODE REVIEW

File: code/game/machinery/doors/firedoor.dm [hunk 1]
```diff
@@ -154,12 +154,18 @@
 	. = TRUE
 	if(!I.tool_use_check(user, 0))
 		return
-	WELDER_ATTEMPT_WELD_MESSAGE
+	if(!welded)
+		WELDER_ATTEMPT_WELD_MESSAGE
+	else
+		WELDER_ATTEMPT_UNWELD_MESSAGE
```
Comment (AffectedArc07) [inline code/game/machinery/doors/firedoor.dm:160]:
```suggestion
	if(welded)
		WELDER_ATTEMPT_UNWELD_MESSAGE
	else
		WELDER_ATTEMPT_WELD_MESSAGE
```

File: code/game/machinery/doors/firedoor.dm [hunk 2]
```diff
@@ -154,12 +154,18 @@
...
 	if(!I.use_tool(src, user, 40, volume = I.tool_volume))
 		return
 	if(!density) //In case someone opens it while it's getting welded
 		return
-	WELDER_WELD_SUCCESS_MESSAGE
+	if(!welded)
+		WELDER_WELD_SUCCESS_MESSAGE
+	else
+		WELDER_UNWELD_SUCCESS_MESSAGE
```
Comment (AffectedArc07) [inline code/game/machinery/doors/firedoor.dm:168]:
```suggestion
	if(welded)
		WELDER_UNWELD_SUCCESS_MESSAGE
	else
		WELDER_WELD_SUCCESS_MESSAGE
```

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]
Cleaner formatting

File: code/game/machinery/doors/firedoor.dm [hunk 3]
```diff
@@ -154,12 +154,18 @@
...
+		WELDER_ATTEMPT_WELD_MESSAGE
 	if(!I.use_tool(src, user, 40, volume = I.tool_volume))
 		return
 	if(!density) //In case someone opens it while it's getting welded
 		return
-	WELDER_WELD_SUCCESS_MESSAGE
+	if(welded)
+		WELDER_UNWELD_SUCCESS_MESSAGE
+	else
+		WELDER_WELD_SUCCESS_MESSAGE
```
Comment (AffectedArc07) [inline code/game/machinery/doors/firedoor.dm:168]:
```suggestion
	if(welded)
		WELDER_WELD_SUCCESS_MESSAGE
	else
		WELDER_UNWELD_SUCCESS_MESSAGE
```

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]
This order seems more correct?

File: code/game/machinery/doors/firedoor.dm (hunk 3 above)
Comment (Vi3trice) [reply code/game/machinery/doors/firedoor.dm:168]: `welded = !welded` should be moved before the success messages for it to make sense.

Comment (AffectedArc07) [reply code/game/machinery/doors/firedoor.dm:168]: ^ that is what I was trying to say, I thought it was above anyway

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**